gallery_app/lib/configs/routes/routes_config.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:gallery_app/modules/button/screens/button_screen.dart';
+import 'package:gallery_app/modules/container_decoration/screens/container_decoration_screen.dart';
+import 'package:gallery_app/modules/cupertino/screens/cupertino_screen.dart';
+import 'package:gallery_app/modules/customboxshape/screens/customboxshape_screen.dart';
+import 'package:gallery_app/modules/home/screens/home_screen.dart';
+import 'package:gallery_app/modules/image_titile_app_bar/screens/image_title_app_bar_screen.dart';
+import 'package:gallery_app/modules/row_column_expanded/screens/row_column_expanded_screen.dart';
+import 'package:gallery_app/modules/scaffold_bottomappbar_actionfloatbutton/screens/scaffold_bottomappbar_actionfloatbutton_screen.dart';
+import 'package:gallery_app/modules/stack_align/screens/stack_align_screen.dart';
+import 'package:gallery_app/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart';
+import 'package:gallery_app/modules/textfield/screens/textfield_screen.dart';
+import 'package:gallery_app/modules/wrap_chip/screens/wrap_chip_screen.dart';
+
+Route<dynamic>? onGenerateRoute(RouteSettings setting) {
+  switch (setting.name) {
+    case "/":
```

Comment:
Create a RouteNames class to contain all the route names of the screens. When used, call from that class to use. When using any route name, it is taken from that class